### DISCUSSION TITLE
Icicle Kit Case

### DISCUSSION THREAD

**Discussion by hsuenaga** [polarfire-soc/polarfire-soc-documentation#193] [Reference designs, FPGA and development kits] [answered]
Hi, I share STL files of ICICLE Kit case. If you looking for case, this may help you. You can print STL files with 3D printers.

The case has access port for: 3 micro USBs, 2 ethernets, SD card, DC input. other ports/sockets are disabled. 
You need 10 M2.5x8mm countersunk head screws to tighten icicle kit and 4 M2.5/M2.6x6mm countersunk head screws to tighten lid.
Probably, different length (< 10mm) screws are OK. I recommend to glue M2.5/M2.6 nuts to lid of the case.

The case is designed using FreeCAD. I share a FreeCAD file too.

Thank you
[ICICLE_BOX.zip](https://github.com/polarfire-soc/polarfire-soc-documentation/files/9430034/ICICLE_BOX.zip)
.

**Comment by AntonKrug** [comment, marked as answer]
Great job, probably you want to share it on sites as thingiverse as well.

Wondering, would you be able to generate STL files without the RISC-V text? 

![image](https://user-images.githubusercontent.com/30338596/186896802-ea0a500b-2643-4235-a3f0-2dcf7d4a8a35.png)

As it currently will cause excesive use of support matterial while without text the whole face could be flat againts the print bed.

**Reply by TommyMurphyTM1234** [reply]
Might additional active or passive cooling be required for the FPGA, power circuits, other components etc. when the board is enclosed in a case?

**Reply by hsuenaga** [reply]
I generated no text version.
https://github.com/hsuenaga/ICICLE_BOX/releases/tag/v1.0.1

**Reply by hsuenaga** [reply]
Cooling is difficult problem. I will measure thermal condition with enclosure and without enclosure for comparison.

**Reply by AntonKrug** [reply]
Looks great, now the first layer has solid foundation againts the print bed.

![image](https://user-images.githubusercontent.com/30338596/187077819-40cf0f56-d377-4b65-9249-465e2c45c03e.png)

**Reply by hsuenaga** [reply]
I measured temperature of SoC using external probe. This is not a junction temperature, just a reference. Probably, we can probe more accurate temperature using TVS, but I don't know how to access it from Linux environment at this time.

I used stress(1) as simple software load. 

![IMG_6934](https://user-images.githubusercontent.com/417037/187873523-076340a0-d1fb-408b-9a51-eb5f2b99a4cf.jpeg)
![FLIR00050](https://user-images.githubusercontent.com/417037/187874186-5768051e-5219-4b2d-8c26-6f5efb916038.png)
![Rplot](https://user-images.githubusercontent.com/417037/187873638-faa5c2d7-c37b-4868-94e8-ed74c17a161a.png)

**Reply by TommyMurphyTM1234** [reply]
So the case seems to be adding c. 10°C to the temperature?

**Reply by AntonKrug** [reply]
It might be even more, as the 'lid open' control was already with closed bottom, might not have as much impact as closed top, but might have a impact.

**Reply by hsuenaga** [reply]
At least 10[C] is added(note that the temperature is not saturated within 14:00-15:00). PolarFire's recommended operating temperature is Tj=100[C] and absolute maximum is Tj=135[C]. Though I didn't measure Tj directly, the temperature is not so high.

Hot spot appeared in FLIR wasn't PolarFire, it was Vitesse VSC8662 GbE phy regardless of link status... but I didn't probe it. I will probe it later if I have time to do.

**Reply by AntonKrug** [reply]
Shiny metal surfaces can't be measured directly with FLIR, they are to IR something like mirrors are to visible light.

https://www.youtube.com/watch?v=lfi5MOhgRbM
